Client/game_sa/CPoolSAInterface.h

```diff
@@ -114,8 +114,9 @@ class CPoolSAInterface
 
     void Delete(uint index) { Release(index); }
 
-    bool IsEmpty(std::int32_t objectIndex) { return m_byteMap[objectIndex].bEmpty; }
-    bool IsContains(uint index)
+    std::int32_t Size() const noexcept { return m_nSize; };
+    bool IsEmpty(std::int32_t objectIndex) const { return m_byteMap[objectIndex].bEmpty; }
```

Comment:
`std::uint32_t`?
also assuming `m_byteMap[objectIndex]` is valid (assert could be used here), `noexcept` can be used here

Reply:
Can the index ever be negative?